ext/php8/ddtrace.c

```diff
@@ -433,9 +432,15 @@ static PHP_RINIT_FUNCTION(ddtrace) {
         DDTRACE_G(disable) = 1;
     }
 
+    // ZAI config is always set up
```

Comment:
The comment is already here @morrisonlevi

Reply:
👍🏻